k9mail/src/main/java/com/fsck/k9/helper/Contacts.java

```diff
@@ -142,6 +142,26 @@ public boolean isInContacts(final String emailAddress) {
     }
 
     /**
+     * Check whether one of the provided addresses belongs to one of the contacts.
+     *
+     * @param addresses The addresses to search in contacts
+     * @return <tt>true</tt>, if one address belongs to a contact.
+     *         <tt>false</tt>, otherwise.
+     */
+    public boolean containsContact(final Address[] addresses) {
```

Comment:
This does almost exactly the same as `isInContacts`, I would suggest a more similar name like `isAnyInContacts`. Could also make it a single method using a variadic signature.

Furthermore, this is quite the expensive method - it's one contacts provider query per email address. This could be collapsed into a single query, and even then it might be a good idea to add an @WorkerThread annotation to discourage its use on the ui thread in the future.